### PR TITLE
Add PHP 8 support and change to github actions

### PR DESCRIPTION
**Pull request by alexander-schranz**
See the result of the github action on my fork:

https://github.com/alexander-schranz/slugifier-api/pull/1/checks

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
```
Comment (dantleech) [inline .github/workflows/test-application.yaml]: do we need to match version numbers here?

Comment (alexander-schranz) [reply .github/workflows/test-application.yaml]: Seems like it only exist master so I removed the version numbers

**Comment by dantleech** [comment]
Looks good to me, does PHP 8 support depend on GH actions? probably we want to migrate everything to GH actions finally cc @dbu

**Comment by alexander-schranz** [comment]
@dantleech travis changed there uses for OSS and there is now just a contingent of free execution minutes and when it ends it really ends no refills. thats why we also migrated sulu to .github actions.

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Install dependencies with Composer
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependencies }}
+                  composer-options: --prefer-dist --no-suggest
```
Comment (dbu) [inline .github/workflows/test-application.yaml]:
```suggestion
                  composer-options: --prefer-dist
```

**Review by dbu — APPROVED** [review]
thanks!

**Comment by dbu** [comment]
https://github.com/symfony-cmf/slugifier-api/releases/tag/2.1.0

**Pull request merged**